experimental/docker-compose/README.md

```diff
@@ -16,13 +16,15 @@ These instructions have been tested on MacOS 10.11.1 and Docker Toolbox 1.9.0b
 
 ## Requirements
 
-Make sure you're happy with the configuration of your spinnaker-local.yml file at ../../config. The instructions to change this are in the file default-spinnaker-local.yml
+Install the [Docker ToolBox](https://www.docker.com/docker-toolbox)
 
-Make sure you have followed the setup needed at the Cloud level from the directions at the root of this project's README file. 
+Navigate to the config directory at the root of this project ( one directory up from the experimental folder ). Copy the `default-spinnaker-local.yml` file into a file called `spinnaker-local.yml`. Edit spinnaker-local.yml to fit your configuration.
 
-Install the [Docker ToolBox](https://www.docker.com/docker-toolbox)
+If you're using aws, create a directory called aws under the config directory. Copy over your credentials from the .aws directory into this directory. If you don't have a .aws directory, follow the instructions here to create those credentials. 
+
+If you're using Google Compute Platform, create a directory called gcp under the config directory. Copy your json credentials into this directory as gcp.json
```

Comment:
The instructions still say to create `aws` and `gcp` dirs under `config` instead of next to it.